website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+## Using dbt-firebolt
+
+### `quote_columns`
+
+To prevent a warning, you should add a configuration as below to your `dbt_project.yml`. For more info, see the [relevant dbt docs page](https://docs.getdbt.com/reference/resource-configs/quote_columns).
+
+
+<File name>
+```yml
```

Comment:
@ericf-firebolt This is not rendering properly on the rich diff in GH, which doesn't necessarily mean it won't render properly on dbt docs, but I just wanted to flag that. Might need some newlines.